src/Bridge/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
 if(ENABLE_QUICKVIEW)
 add_subdirectory_if_module_enabled(VtkGlue)
 endif()
+
+add_subdirectory(NumPy)
```

Comment:
Replace with `add_subdirectory_if_module_enabled` so that this directory is skipped if the user has disabled NumPy support in their ITK build. See [https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/blob/master/src/Core/CMakeLists.txt#L1](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/blob/master/src/Core/CMakeLists.txt#L1) for example usage